uat/custom-components/src/main/java/com/aws/greengrass/utils/LoggingHelper.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.utils;
+
+import ch.qos.logback.classic.Logger;
+
+import java.util.concurrent.TimeUnit;
+
+public final class LoggingHelper {
+    private static double random;
+    public LoggingHelper() {
+    }
+
+    public static void performLogging(Logger logger, int totalLogNumbers, long logWriteFreqSeconds, int logMsgSizeBytes) {
+        for (int i=0; i < totalLogNumbers; i++) {
+            random = Math.random();
+            StringBuilder sb = generateLogMessage(i, logMsgSizeBytes);
+            logger.info(sb.toString());
+            try {
+                TimeUnit.SECONDS.sleep(logWriteFreqSeconds);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    private static StringBuilder generateLogMessage(int seq, int targetMsgSize) {
+        String[] namesArray = {"Jimmy", "Bob", "Amy", "I", "He", "She", "They"};
+        String[] verbsArray = {"play", "study", "develop", "know", "like"};
+        String[] nounsArray = {"code", "math", "football", "computer", "chemistry"};
+
+        int namesIndex = (int)(random * namesArray.length);
+        int verbsIndex = (int)(random * verbsArray.length);
+        int nounsIndex = (int)(random * nounsArray.length);
+        StringBuilder sb = new StringBuilder().
+                append(seq).
+                append(" " + namesArray[namesIndex]).
+                append(" " + verbsArray[verbsIndex]).
+                append(" " + nounsArray[nounsIndex]);
+
+        padLogMessage(sb, targetMsgSize);
+        return sb;
+    }
+
+    private static void padLogMessage(StringBuilder sb, int targetSize) {
```

Comment:
What is the intent behind padding the log messages? What sort of tests will it enable?. I guess it is so that we can test the disk space thread?.

Reply:
My target logic of verifying data loss or duplicate in UAT scenario will be based on the number of log message we received. Thus, by fixing the size of each log message, I can always calculate the expected number of log msgs when given file size, then further compare it with what we receive on CW.

Reply:
I just think we should write logs like ` <seq> - <logger name> some random message`, then we just get all the logs from the log group on cloudwatch and assert there is not a missing sequence number for that component. That way we can know it uploaded everything and also keep it simple.

The only reason I see this would be useful is for testing the space management which is something we will get rid of in a later releasee